src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -0,0 +1,250 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ==============================
+# Schema for:
+#  * 'show cts environment-data'
+# ==============================
+class ShowCtsEnvironmentDataSchema(MetaParser):
+    """Schema for show cts environment-data."""
+
+    schema = {
+        "cts_env": {
+          "current_state": str,
+          "last_status": str,
+          Optional("sgt_tags"): str,
+          Optional("tag_status"): str,
+          Optional("server_list_name"): str,
+          Optional("server_count"): int,
+          Optional("servers"): list
```

Comment:
please adjust to be a key, and get server info

Reply:
This has been updated.